bravado/requests_client.py

```diff
@@ -185,18 +218,55 @@ def check_for_exceptions(self, response):
         except Exception as e:
             add_response_detail_to_errors(e)
 
+    def build_timeout(self, result_timeout):
+        """
+        Build the appropriate timeout object to pass to `session.send(...)`.
+
+        :param result_timeout: timeout that was passed into `result(..)`.
+        :return: timeout
+        :rtype: float or tuple(connect_timeout, timeout)
+        """
+        timeout = None
+        has_service_timeout = 'timeout' in self.misc_options
+        has_result_timeout = result_timeout is not None
+
+        service_timeout = self.misc_options.get('timeout')
+        if has_service_timeout and has_result_timeout:
```

Comment:
Granted, setting it to zero is pretty much useless but I do prefer the readability and consistency of the `has_*` usage in this method.